Gradle.md

```diff
@@ -8,6 +8,8 @@ There are currently five experiments for Gradle. You could also perform these ex
 
 ## Requirements
 
+You must have [Develocity](https://gradle.com/develocity/) in order to use the Develocity Build Validation Scripts. At a minimum, your Develocity user must have the permissions `Publish build scans` and `View build scans and build data` granted explicitly or anonymously, as analyzing the results relies on the use of build scans.
```

Comment:
How come View build scans and build data is required?

Reply:
I understand that you need that permission to view / compare build scans, but strictly speaking, they are not required to use the BVS, e.g., if all you care is whether there were cache misses.

Reply:
I see your point.

You must have one of: `View build scans and build data` or `Access build data via the API`

Having neither and running the scripts would be pointless. I'll try to clarify all this.